rules/integrations/aws/privilege_escalation_attached_malicious_lambda_layer.toml

```diff
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2021/10/03"
+maturity = "production"
+updated_date = "2021/10/03"
+integration = "aws"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when an user attached a Lambda layer to an existing function to override a library that is in use by the 
+function, where their malicious code could utilize the function's IAM role for AWS API calls. This would give an 
+adversary access to the privileges associated with the Lambda service role that is attached to that function.
```

Comment:
```suggestion
Identifies when a user attaches a Lambda layer to an existing function to override a library that is in use by the
function. Attackers can use the Lambda layer to escalate privileges executing API calls using the functions IAM role. 
This would give an adversary access to the privileges associated with the Lambda service role attached to that function.
```